routes/favicon.coffee

```diff
@@ -0,0 +1,27 @@
+request = require 'request'
+fs = require 'fs'
+tmpdir = require 'os'
+	.tmpdir()
```

Comment:
This tried to save some function calls to the `tmpdir` function. But for the sake of consistence, I changed it to multiple function calls.